docs/concepts/vega-chain/data-nodes.md

```diff
@@ -100,16 +100,16 @@ The data node code's configuration includes a set of default retention time fram
 
 Whenever a data node starts up, the existing retention policy for each data type is output in the logs.
 
-### Data retention modes
-When starting a data node, your chosen retention mode will be applied. After initialisation, it's possible to change the retention mode, for example from standard to lite, by changing the retention period setting in the configuration file.
+### Data retention profiles
+When starting a data node, your chosen retention profile will be applied. After initialisation, it's possible to change the retention profile by changing the retention period setting in the configuration file.
 
-The three retention policies are: 
+The three retention profiles are: 
 
-* **Standard (default)**: The node retains data according to the default retention policies (below) of the data node
-* **Archive**: The node retains all data
-* **Lite**: The default retention policy is one day
+* **Archive (default)**: The node retains all data and is the expected and only recommended retention profile for a public data node.
+* **Minimal**: The node retains only data about a network's current state. This can be useful for private data nodes who may want to only serve live data and stream changing states.
+* **Conservative**: The node does not retain all data and per-table rentention is set based on the expected table sizes. This can be useful for private data nodes who may want a customise their per-table data rentention based on their specific usecase.
```

Comment:
"per-table retention is set based on the expected table sizes" - this doesnt mean much to me, could you elaborate on what this would mean to a user or what they can do with this information? Could it just say "is set based on the defaults below"? or is it trying to say more than that?